src/analyses/apron/elinaAnalysis.elina.ml

```diff
@@ -0,0 +1,43 @@
+(** {{!RelationAnalysis} Relational integer value analysis} using {!Elina} domains ([elina]). *)
+
+open Analyses
+
+include RelationAnalysis
+
+let spec_module: (module MCPSpec) Lazy.t =
+  lazy (
+    let module RelImpl = (val ApronDomain.get_implementation "elina") in
+    let module Man = (val ApronDomain.get_manager (module RelImpl)) in
+    let module AD = ApronDomain.D2 (Man) in
+    let diff_box = GobConfig.get_bool "ana.apron.invariant.diff-box" in
+    let module AD = (val if diff_box then (module ApronDomain.BoxProd (AD): RelationDomain.RD) else (module AD)) in
+    let module Priv = (val RelationPriv.get_priv ()) in
+    let module Spec =
+    struct
+      include SpecFunctor (Priv) (AD) (ApronPrecCompareUtil.Util)
+      let name () = "elina"
+    end
+    in
+    (module Spec)

```

Comment:
Could one deduplicate some of this with similar code in the `ApronAnalysis`?

Reply:
I've added a RelationalAnalysis which has the duplicated code from ApronAnalysis and ElinaAnalysis